wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/bootstrap/AbstractBootstrapper.java

```diff
@@ -86,6 +86,9 @@ public void bootstrap() throws InfrastructureException {
         if (finishEventFuture != null) {
             throw new IllegalStateException("Bootstrap method must be called only once.");
         }
+        if (!hasInstallersToBootstrap()) {
```

Comment:
WDYT about moving such logic to InternalRuntime and do not bootstrap at all if installers are missed